qa/workunits/rgw/test_rgw_s3_mp_reupload.py

```diff
@@ -0,0 +1,86 @@
+import boto3
+import sys
+import os
+import subprocess
+
+#boto3.set_stream_logger(name='botocore')
+
+rgw_host = os.environ['RGW_HOST']
+rgw_port = int(os.environ['RGW_PORT'])
+access_key = os.environ['RGW_ACCESS_KEY']
+secret_key = os.environ['RGW_SECRET_KEY']
+
+endpoint='http://%s:%d' % (rgw_host, rgw_port)
```

Comment:
half of the rgw/verify suite should be running radosgw on port 443 and need `https` here